railties/CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+*   Credentials commands (e.g. `bin/rails credentials:edit`) now respect
+    `config.credentials.content_path` and `config.credentials.key_path` when set
+    in `config/application.rb`.
+
+    Before:
+
+      * `bin/rails credentials:edit` ignored `RAILS_ENV`, and would always edit
+        `config/credentials.yml.enc`.
+
+      * `bin/rails credentials:edit --environment foo` would create and edit
+        `config/credentials/foo.yml.enc`.
```

Comment:
```suggestion
      * `bin/rails credentials:edit --environment production` would create and edit
        `config/credentials/production.yml.enc`.
```

I think the changelog is more easily readable by using a realistic example here.

Reply:
I disagree.  Using an obvious placeholder like `foo` makes it clear that we are not just talking about the `production` environment.